minecraft_servers/minergames/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "server_name": "minergames",
+  "nice_name": "MinerGames",
+  "direct_ip": "MinerGames.net",
+  "server_wildcards": [
+    "%.minergames.net"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://www.minergames.net",
+    "web_shop": "https://store.minergames.net",
+    "twitter": "MinerGamesNET",
+    "tiktok": "MinerGames.net",
+    "instagram": "MinerGamesNET",
+    "discord": "https://discord.gg/eUYRxfn4Hg",
+    "youtube": "https://www.youtube.com/channel/UC0mvSh0PBazwyANGSov0Rtg"
+  },
+  "gamemodes": {
+    "citybuild": {
+      "name": "CityBuild",
+      "color": "#55FF55",
+      "url": "https://www.minergames.com/game/citybuild",
+      "command": "/join citybuild"
+    },
+    "duels": {
+      "name": "Duels",
+      "color": "#FF5555",
+      "url": "https://www.minergames.com/game/duels",
+      "command": "/join duels"
+    },
+    "gungame": {
+      "name": "GunGame",
+      "color": "#FFAA00",
+      "url": "https://www.minergames.com/game/gungame",
+      "command": "/join gungame"
+    },
+    "bedwars": {
+      "name": "Bedwars",
+      "color": "#AA0000",
+      "url": "https://www.minergames.com/game/gungame",
+      "command": "/join bedwars"
+    }
+  },
+  "discord": {
+    "server_id": 816762698275618894,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

Reply:
I have compressed the images and set it to false.